gw_spaceheat/actors/home_alone.py

```diff
@@ -146,8 +148,7 @@ def __init__(self, name: str, services: ServicesInterface):
         # Get the weather forecast
         self.weather = None
         self.coldest_oat_by_month = [-3, -7, 1, 21, 30, 31, 46, 47, 28, 24, 16, 0]
-        self.get_weather()
```

Comment:
Remove interactive work from constructor so we can have a constructor which doesn't 'do' things (for example in show_layout.py)